src/plus/gk/checkin.ts

```diff
@@ -66,9 +72,7 @@ export function getSubscriptionFromCheckIn(
 
 	let effectiveLicenses = Object.entries(data.licenses.effectiveLicenses) as [GKLicenseType, GKLicense][];
 	let paidLicenses = Object.entries(data.licenses.paidLicenses) as [GKLicenseType, GKLicense][];
-	paidLicenses = paidLicenses.filter(
-		license => license[1].latestStatus !== 'expired' && license[1].latestStatus !== 'cancelled',
-	);
+	paidLicenses = paidLicenses.filter(license => license[1].latestStatus !== 'cancelled');
```

Comment:
I could be wrong, but I *think* that `cancelled` is Chargebee's version of `expired`, so we may want to remove both of these conditions from the filter (i.e. get rid of this line entirely, as long as `computeSubscriptionState` can ultimately get us the right license state.

Reply:
Didn't know that, I'll remove that as well